src/firecracker/src/main.rs

```diff
@@ -144,6 +144,14 @@ fn main() {
                     process::exit(i32::from(vmm::FC_EXIT_CODE_OK));
                 }
             }
+
+            if let Some(version) = arg_parser.arguments().value_as_bool("version") {
```

Comment:
Since this is fixing a bug (version was previously available in Firecracker), we also need a changelog entry under `# Fixed` section.

Reply:
okay. update asap. Since if this is fixing a bug, could we back-port it to v0.21.0? 😊

Reply:
I think it makes sense. @firecracker-microvm/compute-capsule what do you think?

Reply:
:+1: